catalog-info.yaml

```diff
@@ -0,0 +1,25 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Location
+metadata:
+  name: fxa-monorepo
+spec:
+  type: url
```

Comment:
I am not familiar with backstage yet. Is the `url` the right type given the targets below are paths? How does that resolution work?

Reply:
Hah, no, its not. Good catch!

Reply:
https://backstage.io/docs/features/software-catalog/descriptor-format#spectype-optional documents it, but even though its 'url', that does work for relative paths locally.